### PR TITLE
[MODULAR] Kinkmates are now actually lewd

### PR DESCRIPTION
**Pull request by NullDagaf**
## About The Pull Request

Okay, who's idea was to rename kinkmates to "Dorms-Time Vending Machines"
This reverts it back to Kinkmates, adjusts the description to be actually lewd, and changes the slogans to be.. actually lewd. (tildas are lewd don't @ me)

## Why It's Good For The Game

It's a vending machine selling literal aphrodisiacs. Don't try to sugarcoat it's existence.

## Changelog
:cl:
spellcheck: fixed the misspelling of "KinkMate" from "Dorms-Time Vending Machines"
/:cl:

### CODE REVIEW

**Comment by MLGTASTICa** [comment]
I'd rather see it named as dorms-time and not kink-mate

File: modular_skyrat/modules/nsfw/vendors/code/naughty_vendor.dm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /obj/machinery/vending/dorms
-	name = "\improper Dorms-Time Vending Machine"
-	desc = "A vending machine for all your fun games and activities in the recreational room!"
-	product_ads = "Hey you, yeah you... wanna take a look at my collection?;Come on, take a look!;Remember, always adhere to Nanotrasen corporate policy!"
+	name = "KinkMate"
+	desc = "A vending machine for all your unmentionable desires."
+	product_ads = "Hey you, yeah you... wanna take a look at my collection~?;Kinky!;Sexy!;Check me out, big boy~"
```
Comment (FlamingLily) [inline modular_skyrat/modules/nsfw/vendors/code/naughty_vendor.dm]: ?~ is more visually appealing than ~?, In my opinion. A minor thing, but this is a minor change.

Comment (NullDagaf) [reply modular_skyrat/modules/nsfw/vendors/code/naughty_vendor.dm]: Wasn't sure myself. Any case, changed.

**Comment by NullDagaf** [comment]
don't worry
the integration error is just monke

**Comment by YakumoChen** [comment]
Are kinkmates actually mapped right now? I haven't seen them at all.
E: I saw one on Delta but I have a hard time finding them on other maps

**Comment by Gandalf2k15** [comment]
Ambiguity good.

**Comment by Weblure** [comment]
> Ambiguity good.

The ambiguous machine with pictures of dildos on it that sells sex apparel and aphrodisiacs... nice.
Doesn't have to be "KinkMate", but "Dorms-Time" sounds like a name made up by a literal child and should be changed.

**Pull request closed without merging**